### ISSUE TITLE
Prompt cursor position differs for multiline questions

### ISSUE THREAD

**Issue by thymikee** [enquirer/enquirer#268]
## Bug Report

When the `.prompt()` question is 2 or 3 lines long, there's extra 1 or 2 newlines added to the cursor position after selecting an answer:

<img width="1005" alt="Screen Shot 2020-03-16 at 13 41 54" src="https://user-images.githubusercontent.com/5106466/76759804-b6500000-678c-11ea-92d4-2bef3b98435b.png">

## Expected behavior

I'd expect it to behave the same when question is 1 line:

![image](https://user-images.githubusercontent.com/5106466/76759769-a46e5d00-678c-11ea-9fe8-3b926e31eed5.png)

## Reproduction steps

See this repo: https://github.com/thymikee/enquirer-cursor-repro

## Context

https://github.com/react-native-community/cli/pull/1010

**Comment by doowb** [comment]
@thymikee thank you for the reproduction steps!

It looks to be just be extra newlines. From the description and the screen shots in the other issue, I originally thought there might be a different issue.

I'll try to take a look soon.

**Comment by jonschlinkert** [comment]
This is fixed in the next version, we can look and see if there is quick patch for the current version, but it might be better to wait since we're almost ready to publish a major.

**Comment by thymikee** [comment]
@jonschlinkert sounds reasonable. Is there an alpha/beta version I could test? Haven't seen an npm tag for it.

**Comment by jonschlinkert** [comment]
> Is there an alpha/beta version I could test? Haven't seen an npm tag for it.

No, I'm sorry to say--with rare exception--we don't work on in-progress code on public repositories anymore.

**Comment by thymikee** [comment]
Sure, that's understandable. Thanks!

**Comment by brodybits** [comment]
> we can look and see if there is quick patch for the current version, but it might be better to wait since we're almost ready to publish a major

Any idea when we can expect a new version with this issue fixed?

This seems to be really holding up <https://github.com/react-native-community/cli/pull/1010>.

**Comment by jonschlinkert** [comment]
> This seems to be really holding up [react-native-community/cli#1010](https://github.com/react-native-community/cli/pull/1010).

Good to know, thanks. 

> EDIT: Is there anything we can do from the user community to help get the new version with this issue fixed?

A PR would be great!

**Comment by brodybits** [comment]
> See this repo: https://github.com/thymikee/enquirer-cursor-repro

Hey I tried running the repo and am not able to see the extra newlines. I am on macOS Catalina in case it makes any difference.

**Comment by jonschlinkert** [comment]
> Hey I tried running the repo and am not able to see the extra newlines. I am on macOS Catalina in case it makes any difference.

Yeah, although I mentioned that "it's fixed" in the next version, I'm betting it will still happen with certain terminals. It would be awesome if people could list the terminals they use, and whether or not they are seeing this bug. Thanks

**Comment by thymikee** [comment]
FYI, the terminal I used was iTerm2. I observed the same behavior on default macOS Terminal app as well.
